### ISSUE TITLE
Make Mumble client not overwrite the Mumble.conf file when run

### ISSUE THREAD

**Issue by asheeshr** [mumble-voip/mumble#3904]
Is there a way to make the client use the Mumble.conf file in read only mode or ensure that the file does not get overwritten by the client?

Making the file permissions root only to force this causes other issues with the client.

**Comment by Krzmbrzl** [comment]
I don't think so. As far as I have seen there is no such mechanism in place. If something in Mumble's configuration changes, the config will be updated. (I could have missed something though. So don't take this as the ultimate truth :P)

What use case do you have for this? Isn't the config specifically meant to change in order for Mumble to restore the state it left off from? :thinking:

**Comment by asheeshr** [comment]
The use case is to be able to share a standard configuration with custom information (comments) and prevent users from changing any parameters when using the client.

**Comment by Kissaki** [comment]
AFAIK we make use of Qt functionality for transparent configuration handling so this may not be easy to do from our consumer side of that. So the first step would be checking on that, how we use the Qt functionality and what options the API provides in that regard.
